Topics/Product_Management.md

```diff
@@ -1 +1,39 @@
 ## Resources for Product Management
+
+### What does product manager do?
+There is a [guide on what the product manager does](https://www.productplan.com/learn/what-is-product-management/), and one of the introductory and key concepts has been extracted as follows:
+
+Most product professionals spend the majority of their time focused on the following:
+* __**Conducting Research:**__ Researching to gain expertise about the company’s market, user personas, and competitors.
+* __**Developing Strategy:**__ Shaping the industry knowledge they’ve learned into a high-level strategic plan for their product—including goals and objectives, a broad-strokes overview of the product itself, and maybe a rough timeline.
+* __**Communicating Plans:**__ Developing a working strategic plan using a product roadmap and presenting it to key stakeholders across their organization: executives, investors, development teams, etc. Ongoing communication across their cross-functional teams throughout the development process and beyond.
+* __**Coordinating Development:**__ Assuming they have received a green light to move forward with their product’s strategic plan, coordinate with the relevant teams—product marketing, development, etc.—to begin executing the plan.
+* __**Acting on Feedback and Data Analysis:**__ Finally, after building, testing, and introducing the product to the marketplace, learning via data analysis and soliciting direct feedback from users, what works, what doesn’t, and what to add. Working with the relevant teams to incorporate this feedback into future product iterations.
+
+### How should CSC301 students perform product management?
```

Comment:
Very useful insight for CSC301 students, great job! The instructions for utilizing product management strategies are neatly tied with the above info on what product managers do. It would be nice to also point to David's lecture from Week 2 on product management.